src/Shared/Recruit.Vacancies.Client/Domain/Entities/Vacancy.cs

```diff
@@ -69,11 +69,20 @@ public class Vacancy
                                   Status == VacancyStatus.Referred)
                                  && IsDeleted == false;
         /// <summary>
-        /// We can only edit draft & referred vacancies that have not been deleted
+        /// We can only edit draft , referred vacancies that have not been deleted
```

Comment:
why change the '&' to ','?

Reply:
changed the text to add Review status. as we have decided to use CanEmployerEdit ..  will undo the text changes

Reply:
done